src/abstractdataframe/abstractdataframe.jl

```diff
@@ -430,6 +430,11 @@ function StatsBase.describe(df::AbstractDataFrame; stats::Union{Symbol,AbstractV
         stats = allowed_fields 
     end
 
+    if size(df, 1) == 0
+        stats = [:nmissing, :eltype]
+        warn("This DataFrame has 0 rows, so describe output has been truncated")
```

Comment:
The reason for the error is that the user might have specified `stats = [:mean, :min, :max]` but we give them two unrelated statistics. Perhaps we should just print and say something?

Reply:
Ah - now I see! But I would rather produce:
* `nothing` if type of the column does not allow calculation of the quantity or we are unsure if it does
* `missing` if it does allow it (in practice `<:Union{Missing, Real}`)

However I am not sure what is best. But in general I guess we have moved `describe` to produce `DataFrame` to allow for programmatic use of the result of `describe`. If so - then a warning is not very useful, as someone might later assume in the code that column e.g. for `min` is produced if it were requested.

Reply:
Good point! Returning a `nothing` is more consistent with the overall style too. 

I think we should avoid `missing` since we return `nothing` for everything else (and printing is prettier).

Reply:
OK with me.